### DISCUSSION TITLE
[ UPDATED GUIDE ] Guide to install FastSD CPU on Termux Android!

### DISCUSSION THREAD

**Discussion by Nick088Official** [rupeshs/fastsdcpu#278] [Show and tell]
[Here](https://github.com/rupeshs/fastsdcpu?tab=readme-ov-file#android) I seen there were 2 termux guides, one using the official proot-distro ubuntu and the other using the 3rd party ubuntu-in-termux.

The issue is that both of those have recently switched from ubuntu 22.04 to 24, which makes so you can't natively install python 3.10 to run the program, and would need to do other steps to use it.

So, I made an updated guide which uses the ubuntu-oldlts from the official proot-distro, being able to still use ubuntu 22.04, even if it there are some differences other than that. I tried to make it as easy as I can while also explaining the users what they are doing.

## You can check it out [HERE](https://docs.google.com/document/d/1gQLQz7GIzvh2G6BtJOnbxUQfERATUw8q4wLAKGjBPVk/edit?usp=sharing)

**Comment by MikiBiH75000** [comment]
Kannst du mir helfen 
: dpkg was interrupted, you must manually run 'dpkg --configure -a' to correct the problem.

**Reply by Nick088Official** [reply]
I don't know german so I just used a translator,
Can you elaborate more your issue and on which step it happens?

**Reply by MikiBiH75000** [reply]
~ $ pkg install python
No mirror or mirror group selected. You might want to select one by running 'termux-change-repo'
Checking availability of current mirror:
[*] https://ftp.agdsn.de/termux/termux-main: ok
E: dpkg was interrupted, you must manually run 'dpkg --configure -a' to correct the problem.
~ $

**Reply by Nick088Official** [reply]
> ~ $ pkg install python No mirror or mirror group selected. You might want to select one by running 'termux-change-repo' Checking availability of current mirror: [*] https://ftp.agdsn.de/termux/termux-main: ok E: dpkg was interrupted, you must manually run 'dpkg --configure -a' to correct the problem. ~ $

Are you sure you installed termux from Fdroid? It's better you reinstall it from that, also be sure to follow the guide, it doesn't say to run `pkg install python`

![image](https://github.com/user-attachments/assets/4702b082-ed20-400c-a3d1-5e6ac0c625b8)

**Reply by MikiBiH75000** [reply]
I ned help !!
I have installed Kali and in termux the white bar is not moving. As soon as I press sudo apt install, it asks me for my password and then I can't enter the password because the white bar is not responding. Can you help me? Thanks. Mike

**Reply by Nick088Official** [reply]
> I ned help !! I have installed Kali and in termux the white bar is not moving. As soon as I press sudo apt install, it asks me for my password and then I can't enter the password because the white bar is not responding. Can you help me? Thanks. Mike

I haven't used Kali, in the guide it's explained how to use it via Ubuntu
Also, you need to ROOT your phone to use Sudo on Termux, you can still apt install things without termux (as far as I have tested on Ubuntu)
Maybe try without sudo or follow it in Ubuntu as in the Guide

**Reply by MikiBiH75000** [reply]
I have kali linux on laptop install

**Reply by Nick088Official** [reply]
Oh that's different than termux, I'm guessing the password depends on how you set your PC, maybe try cehcking https://github.com/rupeshs/fastsdcpu?tab=readme-ov-file#fastsd-cpu-on-linux

**Comment by rupeshs** [comment]
@Nick088Official  Thanks
Starting from v1.0.0-beta.200 we will use uv for installation
Updated installation guide : https://nolowiz.com/how-to-install-and-run-fastsd-cpu-on-android-temux-step-by-step-guide/
Release : https://github.com/rupeshs/fastsdcpu/releases/tag/v1.0.0-beta.200
